File: app/src/main/java/com/artmurka/artmurkaapp/model/pojo/itemlist/novaposhta/WarehousesRequest/MethodProperties.java

package com.artmurka.artmurkaapp.model.pojo.itemlist.novaposhta.WarehousesRequest;

import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

public class MethodProperties {

    @SerializedName("CityRef")
    @Expose
    private String settlementRef;

    public MethodProperties(String settlementRef){
        this.settlementRef = settlementRef;
    }

    public String getSettlementRef() {
        return settlementRef;
    }

    public void setSettlementRef(String settlementRef) {
        this.settlementRef = settlementRef;
    }

    public MethodProperties withSettlementRef(String settlementRef) {
        this.settlementRef = settlementRef;
        return this;
    }

}

File: app/src/main/java/com/artmurka/artmurkaapp/data/model/pojo/itemlist/novaposhta/WarehousesRequest/MethodProperties.kt
package com.artmurka.artmurkaapp.data.model.pojo.itemlist.novaposhta.WarehousesRequest

import com.google.gson.annotations.Expose
import com.google.gson.annotations.SerializedName

class MethodProperties(@field:SerializedName("CityRef")
                       @field:Expose
                       var settlementRef: String?) {

    fun withSettlementRef(settlementRef: String): MethodProperties {
        this.settlementRef = settlementRef
        return this
    }

}
